src/libutil/ustring.cpp

```diff
@@ -433,12 +428,81 @@ ustring::make_unique(string_view strref)
         strref = string_view("", 0);
 
     size_t hash = Strutil::strhash(strref);
```

Comment:
That's a good point.

Reply:
Sure, I can take a look at that, but since I'd like to backport this current PR to the release branch (so OSL can build against a released OIIO 2.2), I want to defer the size_t issue to a separate PR so that I can merge it into master only, since the type change would create an ABI incompatibility that can't be backported.